Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift

```diff
@@ -64,7 +66,11 @@ final class FluidMenuBarExtraWindow<Content: View>: NSPanel {
         titlebarAppearsTransparent = true
 
         animationBehavior = .none
-        collectionBehavior = [.auxiliary, .stationary, .moveToActiveSpace, .fullScreenAuxiliary]
+        if #available(macOS 13.0, *) {
+            collectionBehavior = [.auxiliary, .stationary, .moveToActiveSpace, .fullScreenAuxiliary]
+        } else {
+            collectionBehavior = [.stationary, .moveToActiveSpace, .fullScreenAuxiliary]
+        }
```

Comment:
Same here, how does this affect behavior (i.e. animations) on macOS 12? Do you have a screen recording of the result?

Reply:
Yup, here ya go

https://github.com/lfroms/fluid-menu-bar-extra/assets/469799/ce0535cd-2a80-4751-b381-265fde2437e0

Reply:
Cool, that seems to work well.